src/styles.ts

```diff
@@ -61,7 +61,7 @@ export default css`
 
   clock-weather-card-forecast-row {
     display: grid;
-    grid-template-columns: 2rem 2rem 2.1rem auto 2.1rem;
+    grid-template-columns: 2.5rem 2rem 2.1rem auto 2.1rem;
```

Comment:
to not waste space if a user uses the regular `daily` instead of `hourly` forecast, i'd propose to make this change conditional to only change the column width from `2rem` to `2.5rem` if `forecast_hourly` is `true`

Reply:
Was especcially needed when supporting the 12h display. Added it.